runtimes/alibi-detect/mlserver_alibi_detect/cd/runtime.py

```diff
@@ -0,0 +1,42 @@
+from mlserver_alibi_detect.runtime import AlibiDetectRuntime
+from mlserver_alibi_detect.errors import InvalidAlibiDetector
+from mlserver.utils import get_model_uri
+from alibi_detect.utils.saving import load_detector
+from importlib import import_module
+from typing import Any
+import pickle
+
+ALIBI_DETECT_DRIFT_MODULE = "alibi_detect.cd"
+
+
+class AlibiDriftDetectRuntime(AlibiDetectRuntime):
+    """
+    Implementation of the MLModel interface to load and serve Alibi drift models.
+    """
+
+    async def load(self) -> bool:
+
+        model_uri = await get_model_uri(self._settings)
+
+        if not bool(self.alibi_detect_settings.init_detector):
+            self._model = load_detector(model_uri)
+        else:
+            detector_data = pickle.load(open(model_uri, "rb"))
+            try:
+                detector_type = self.alibi_detect_settings.detector_type
+
+                detect_module = import_module(ALIBI_DETECT_DRIFT_MODULE)
```

Comment:
Instead of doing the import manually, we could let Pydantic handle that by changing the `detector_type` field type to [`PyObject`](https://pydantic-docs.helpmanual.io/usage/types/#pydantic-types). 

We leverage this already for the `implementation` field:

https://github.com/SeldonIO/MLServer/blob/f088620d67cd366ad19857db38d5f89c7586a427/mlserver/settings.py#L65